generators/entity-server/templates/quarkus/src/test/java/package/domain/EntityTest.java.ejs

```diff
@@ -54,4 +67,43 @@ public class <%= entityClass %>Test {
         assertThat(<%= asEntity(entityInstance) %>1).isNotEqualTo(<%= asEntity(entityInstance) %>2);
     <%_} _%>
     }
+    
+    @Test
+    public void entityAnnotations() throws Exception {
```

Comment:
What's the purpose of this test?
Sounds like a test `ArchTest.java`
I think this kind of "technical testing" should be done in a dedicated class. 
EntityTest should be dedicated to validate common entities behaviors, not their definition.